<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_sim_window</class>
 <widget class="QMainWindow" name="qt_sim_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="handleWidth">
          <number>10</number>
         </property>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="6" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>dt</string>
                 </property>
                </widget>
               </item>
               <item row="10" column="1">
                <widget class="QSpinBox" name="spinBox_d_rc">
                 <property name="maximum">
                  <number>200</number>
                 </property>
                 <property name="value">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_r_inner">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="value">
                  <double>0.700000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_h">
                 <property name="value">
                  <double>2.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>n / r part</string>
                 </property>
                </widget>
               </item>
               <item row="10" column="2">
                <widget class="QSpinBox" name="spinBox_d_rh">
                 <property name="maximum">
                  <number>200</number>
                 </property>
                 <property name="value">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_r_part">
                 <property name="minimum">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>height</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="3">
                <widget class="QPushButton" name="pushButton_reset_sim">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>reset</string>
                 </property>
                </widget>
               </item>
               <item row="9" column="1">
                <widget class="QSpinBox" name="spinBox_d_nc">
                 <property name="minimum">
                  <number>10</number>
                 </property>
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                 <property name="value">
                  <number>200</number>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_grav_x">
                 <property name="minimum">
                  <double>-100.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_dt">
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.010000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="5" column="2">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_grav_y">
                 <property name="minimum">
                  <double>-100.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>-9.810000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSpinBox" name="spinBox_npart">
                 <property name="maximum">
                  <number>1000000</number>
                 </property>
                 <property name="value">
                  <number>1000</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>setup</string>
                 </property>
                </widget>
               </item>
               <item row="10" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>rad c/h</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="3">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_grav_z">
                 <property name="minimum">
                  <double>-100.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>grav x/y/z</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_r_outer">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="value">
                  <double>1.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>diods</string>
                 </property>
                </widget>
               </item>
               <item row="9" column="2">
                <widget class="QSpinBox" name="spinBox_d_nh">
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                 <property name="value">
                  <number>200</number>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>sim</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_d_brightness">
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>500.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="9" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>n c/h</string>
                 </property>
                </widget>
               </item>
               <item row="12" column="0">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>brightness</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="2">
                <widget class="QCheckBox" name="checkBox_active_sim">
                 <property name="text">
                  <string>active</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="8" column="2">
                <widget class="QCheckBox" name="checkBox_active_diodes">
                 <property name="text">
                  <string>active</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>rad o / i</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QLabel" name="label_image">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>NONE</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_scene">
           <item row="0" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
